services/todo-legacy/index.ts

```diff
@@ -0,0 +1,35 @@
+import { Application, Server } from '@loopback/core';
+import { setBindings } from './util/setup';
+
+class TodoApplication extends Application {
+  private _startTime: Date;
+
+  constructor() {
+    super();
+    const app = this;
+    setBindings(app);
+  }
+
+  async start() {
+    this._startTime = new Date();
+    const server = new Server(this, { port: process.env.PORT || 3000 });
```

Comment:
Old proposal, cc @bajtos We need to get rid of the Server stuff soon and use App itself (app.start() as proposed in wiki -- https://github.com/strongloop/loopback-next/wiki/Hello-world)